src/theory/quantifiers/quant_conflict_find.cpp

```diff
@@ -2061,9 +2065,8 @@ void QuantConflictFind::check( Theory::Effort level, unsigned quant_e ) {
         debugPrint("qcf-debug");
         Trace("qcf-debug") << std::endl;
       }
-      short end_e = getMaxQcfEffort();
       bool isConflict = false;
-      for( short e = effort_conflict; e<=end_e; e++ ){
+      for (Effort e : getQcfEfforts()) {
```

Comment:
I don't want to change getQcfEffort to a vector. That is my main complaint about this PR.
Btw, I don't know your preferred way of iterating on enums, but this is what I did in a recent pull request:

https://github.com/CVC4/CVC4/commit/85df7998e4362e0a9c796146d07d7b9e91045a31#diff-d66c6276213ae410fc43f004a2dd20a9R562

I'd like to do something similar here.

Reply:
Removed the getQcfEffort() vector and switched back to a similar way of iterating through the values.